package.json

```diff
@@ -10,18 +10,21 @@
   "dependencies": {},
   "devDependencies": {
     "css-loader": "^0.18.0",
+    "eslint": "^1.6.0",
+    "eslint-loader": "^1.0.0",
```

Comment:
I'd rather avoid adding dependencies (more lock-in to WebPack, more risk on environment update).

What about simply adding `eslint` as a `test` script (or as another script, as [here](https://github.com/sgmap/mes-aides-ui/blob/rewrite/package.json#L25)), and letting each contributor define her own workflow?

Note: remember to update `circle.yml` to add the [linting command](https://github.com/sgmap/mes-aides-ui/blob/rewrite/circle.yml#L41-L47) in CI.